src/portfolio/APIClient.ts

```diff
@@ -96,6 +96,60 @@ export type PositionQty = {
   qty: number;
 };
 
+export type Backtesting = {
+  MDDMaxToMin: number;
+  MDDInitialToMin: number;
+  dateMax: Date;
+  dateMin: Date;
+  maxValue: number;
+  minValue: number;
+  initialValue: number;
+  bestYear: {
+    changeBest: number;
+    yearBest: string;
+    growthRateBest: number;
+  };
+  worstYear: {
+    changeWorst: number;
+    yearWorst: string;
+    growthRateWorst: number;
+  };
+  finalPortfolioBalance: number;
+  CAGR: number;
+  standardDeviation: number;
+  sharpeRatio: number;
+};
+
+// Types describing the JSON response of API calls.
+// The correctness of these types is assumed, no checks are performed.
+
+export type BacktestingResponse = {
+  error: string;
+  success: {
+    MDDMaxToMin: number;
+    MDDInitialToMin: number;
+    dateMax: string;
+    dateMin: string;
+    maxValue: number;
+    minValue: number;
+    initialValue: number;
+    bestYear: {
+      changeBest: number;
+      yearBest: string;
+      growthRateBest: number;
+    };
+    worstYear: {
+      changeWorst: number;
+      yearWorst: string;
+      growthRateWorst: number;
+    };
+    finalPortfolioBalance: number;
+    CAGR: number;
+    standardDeviation: number;
+    sharpeRatio: number;
+  };
+};
```

Comment:
`success` can also be an empty object (if there is an error). This case should be handled. The `error` string is useless for us because it's not localized, but an error should be thrown if it's not empty.